website/components/meta-tags/OgMetaTags.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react"
+
+export type OgMetaTags = {
+  url?: string
+  title?: string
+  description?: string
+  image?: string
+}
+
+const OgMetaTags: React.FC<OgMetaTags> = ({
+  url = "https://altruisto.com",
+  title = "Do good just by shopping online",
+  description = "Install an extension and when you buy something online, people in extreme poverty will get medicines, bed nets, or financial aid.",
+  image = "https://altruisto.com/images/social-media-cover.png"
+}) => (
+  <>
+    {url && <meta property="og:url" content={url} />}
```

Comment:
since we have default values set in destructuring  of props, this condition will never be false